### DISCUSSION TITLE
表单是layout是vertical垂直布局，如何在表单label和输入框中间插入一个React节点呢

### DISCUSSION THREAD

**Discussion by ChenKun1997** [ant-design/ant-design#43533] [Q&A]
如果在label里面写，会影响到必填的符号还有tooltip符号，如下图
![image](https://github.com/ant-design/ant-design/assets/24249578/c70f4024-8c19-4235-bbfe-d7366ef543ad)

**Comment by linxianxi** [comment]
自定义 Input 组件
<img width="513" alt="image" src="https://github.com/ant-design/ant-design/assets/47104575/ba3f5f0e-d7c0-4e8c-990c-803f2560f67f">

**Comment by ChenKun1997** [comment]
> 自定义 Input 组件 <img alt="image" width="513" src="https://user-images.githubusercontent.com/47104575/253231056-ba3f5f0e-d7c0-4e8c-990c-803f2560f67f.png">
Form.Item下面只能放表单元素，不能放其他的，放其他的话，在表单之变更时会报错

**Reply by linxianxi** [reply]
这样就行了，属性都传给 Input 了

**Comment by Yuiai01** [comment]
> > 自定义 Input 组件 <img alt="image" width="513" src="https://user-images.githubusercontent.com/47104575/253231056-ba3f5f0e-d7c0-4e8c-990c-803f2560f67f.png">
> > Form.Item下面只能放表单元素，不能放其他的，放其他的话，在表单之变更时会报错

自定义表单组件需要提供 value 和 onChange 事件，你可以参考这个 https://ant.design/components/form-cn#components-form-demo-customized-form-controls
